coral-trino/src/main/java/com/linkedin/coral/trino/trino2rel/ParseTreeBuilder.java

```diff
@@ -27,16 +54,186 @@
 
 import com.linkedin.coral.common.HiveTypeSystem;
 import com.linkedin.coral.common.calcite.CalciteUtil;
-import com.linkedin.coral.hive.hive2rel.functions.HiveFunctionResolver;
-import com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry;
 
-import coral.shading.io.trino.sql.tree.*;
```

Comment:
Why do we remove it and add so many lines of imports?

Reply:
I think something must have happened when I ran refactoring with the IDE, reverting this back to just `import coral.shading.io.trino.sql.tree.*;`

Reply:
Also reverted all other non-necessary imports.